native-agent.md

```
@@ -8,38 +8,42 @@ Before attempting a first install, you will need to install these dependencies o
 
-By default, the agent will advertise almost all your environment variables, including the username, some key directories, and many other elements. You can get rid of this by setting up the VSO_AGENT_IGNORE environment variable. Run the script with this specific setup to create this environment variable **before** you set up the configuration.
+> [!Important]
+> By default, the agent will advertise almost all your environment variables, including the username, some key directories, and many other elements. You can get rid of this by setting up the VSO_AGENT_IGNORE environment variable. Run the script with this specific setup to create this environment variable **before** you set up the configuration.
 
+Download this latest repository source.
 From your source download directory, open the terminal as administrator, then:
 
+(you may need to change the execeution properties in windowsdeveloper options and/or `Set-ExecutionPolicy Bypass -Scope Process`, ) until we have signed the file
+
+Copy the [capabilities.ps1](./agent/capabilities.ps1) file to the `C:\agents` folder, and then run:
 ```powershell
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)